src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -131,6 +133,53 @@ public static void setUp() throws IOException {
                 .willReturn(aResponse().withBody(jsonFrom("currency/latest.json"))));
     }
 
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromConsumable() throws IOException, JSONException {
+        // given
+        // consumable bid response for imp 001
+        wireMockRule.stubFor(post(urlPathEqualTo("/consumable-exchange"))
+                .withHeader("Cookie", equalTo("azk=CS-UID"))
+                .withHeader("Origin", equalTo("http://www.example.com"))
+                .withHeader("Accept", equalTo("application/json"))
+                .withHeader("User-Agent", equalTo("userAgent"))
+                .withHeader("Forwarded", equalTo("for=192.168.244.1"))
+                .withHeader("Referer", equalTo("http://www.example.com"))
+                .withHeader("X-Forwarded-For", equalTo("192.168.244.1"))
+                .withHeader("Host", equalTo("localhost:8090"))
+                .withHeader("Content-Length", equalTo("302"))
+                .withHeader("Content-Type", equalTo("application/json;charset=UTF-8"))
+                .withCookie("azk", equalTo("CS-UID"))
+                .withRequestBody(matchingJsonPath("$.time"))
+//                .withRequestBody(equalToJson(jsonFrom("openrtb2/consumable/test-consumable-bid-request-1.json")))
```

Comment:
Pls make clean.

Reply:
Done! By the way, should be try to add come additional logic, like adding more matching json params, to request matching or just leave it like this and don't care about bidder request at all since we got all it's logic covered at bidder's unit tests? I'll also remove the `openrtb2/consumable/test-consumable-bid-request-1.json` file, since it is not required.